lib/flat_search/olx_scraper.ex

```diff
@@ -65,12 +64,25 @@ defmodule FlatSearch.OlxScraper do
       negotiation: negotiable?(document),
       surface: get_surface(document),
       photo_links: get_photo_links(document),
-      link: url
+      link: url,
+      region: Enum.at(get_localization(document), 0),
+      city: Enum.at(get_localization(document), 1),
+      district: Enum.at(get_localization(document), 2)
     }
 
     Flats.create_flat(flat_record)
   end
 
+  defp get_localization(document) do
+    # Returns list of strings
+    # [Region, City, District | null] 
```

Comment:
I would put comments above the function defp or even better looking at the content of comment -> look into this
https://hexdocs.pm/elixir/1.12/typespecs.html

Reply:
Btw, should every function by default be with type it returns or just some?

Reply:
What about that?
```
@spec get_localization(document :: HTML) :: [String.t() | nil, ...]
```

Reply:
It depends on the projects, generally, it can be helpful for other programmers(or you in the future) to know the type a specific functions returns.  But it's more like a personal preference when setting up the style guide for projects(since these types aren't enforcing anything)